### PR TITLE
Add remove task

### PR DESCRIPTION
**Pull request by emmanuelsteven**
In this section, I was able to work on these functionalities ;
-Remove all hardcoded items from the tasks array.
-Create a new JavaScript file for the new functionality.
-Implement a function for adding a new task.
-Implement a function for deleting a task.
-Implement a function for editing task descriptions.

### CODE REVIEW

File: src/script.js
Comment (AdnanOlarmmi) [inline src/script.js:1]: mobile

- More information at the bottom 👇  wherever appropriate, 🏃

File: src/addRemove.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+              id="${task.id}"
+              name="task"
+              value="task"
+              ${task.completed ? 'checked' : ''}
+              class="checkbox"
+            />
+            <input
+              type="text" id="${task.id}" class="text-area" name="task" value="${task.description}" />
+              
+              <span class="material-symbols-outlined delete"> delete </span>
```
Comment (AdnanOlarmmi) [inline src/addRemove.js:52]: - [ ] Nice one with the design from the image above. It seems the last item is not deleting except for one of the items before it is deleted, kindly fix that bug 😁. as it is a requirement.

File: src/script.js [hunk 2]
```diff
@@ -1,50 +1,33 @@
...
-  {
-    index: '1',
-    description: 'Mark text script',
-    completed: 'false',
-  },
-
-];
+import TodoList from './addRemove.js';
 
 const taskList = document.querySelector('.task-container');
```
Comment (AdnanOlarmmi) [inline src/script.js:4]: - [ ] Kindly note that the edited text doesn't persist in the `localstorage` after refresh 🙃 as it is a requirement.

File: src/addRemove.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+          initialIndex += 1;
+        });
+      };
+
+      /* The `showCompleted` method is a function of the `TodoList` class that sets the `completed`
+      property of all tasks in the `tasks` array to `true`. This method can be used to mark all
+      tasks as completed at once. */
+      showCompleted = () => {
+        this.tasks.forEach((task) => {
+          task.completed = true;
```
Comment (AdnanOlarmmi) [inline src/addRemove.js:91]: - [ ] Kindly note that the `completed` value doesn't persist in the `localstorage` after refresh. 😃 as it is a requirement.

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @emmanuelsteven, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Nice design.
- Nice implementations.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @emmanuelsteven 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**